src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
+@import './variables';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  width: 726px;
+  box-sizing: border-box;
+  padding: 10px;
```

Comment:
It's a good practice to use variables for all values that are used more than once throughout the code. I see that you have used the padding value of 10px here, and it might be used elsewhere in your other files. Consider moving it to your `variables.scss` file. This will make your code more maintainable as you will only need to change the value in one place should it need to be updated in the future.